test/acceptance/test/add_tests.go

```diff
@@ -673,6 +620,155 @@ var _ = Describe("Weave GitOps Add Tests", func() {
 		})
 	})
 
+	//deployment-type=default k, default h, custom h| repo=private | app status | app list | app-config-url=url
```

Comment:
Nit: If possible comments should be removed and can be added the required info in test It().

Reply:
I added comments to keep track of what flags we're covering for each test. Seemed to noisy for `It` blocks